iOS/DuckDuckGo/LargeOmniBarState.swift

```diff
@@ -213,7 +218,10 @@ struct LargeOmniBarState {
         let showBackground = true
         let showClear = false
         var showAbort: Bool { isLoading }
-        var showRefresh: Bool { !isLoading }
+
+        // To be removed
```

Comment:
Removed once we turn on the flag for everyone?

Reply:
once we know this UI is truly settled - just trying to minimise changes to the omnibar - so it's dead code, but I'll create a follow up to remove it